internal/slack/resource_slack_alert.go

```diff
@@ -99,6 +100,12 @@ func (r *SlackAlertResource) Schema(ctx context.Context, req resource.SchemaRequ
 				Computed:            true,
 				Default:             booldefault.StaticBool(false),
 			},
+			"send_direct_message": schema.BoolAttribute{
+				MarkdownDescription: "If Slack is connected, it will send notifications to the requesting user. Cannot be used in conjunction with 'slack_channel_id'",
```

Comment:
```suggestion
				MarkdownDescription: "If Slack is connected, it will send notifications to the approvers. Cannot be used in conjunction with 'slack_channel_id'",
```